### PR TITLE
#238 | Implement the Admin User Registration Application Detail Endpoint

### PR DESCRIPTION
**Pull request by ibrahimdenizz**
### Description

I have implemented detail endpoint for the admin user registration application

### Related Issue

#238 

### Proposed Changes

Please provide a brief and descriptive title for your pull request, summarizing the changes or features introduced. It
should be in English and clearly indicate the purpose of the pull request. Optionally, you can provide a more detailed
description of what you have done.

- Branches should follow the below `Naming Conventions of Branchs` header in [README.md](https://github.com/afet-yonetim-sistemi/ays-be/edit/main/README.md) File

### Additional Information

If there's any additional information or context that the reviewers should know, please provide it here.

### Checklist

Please check the following before submitting your pull request:

- [X] I have tested my changes locally and they are working as expected.
- [X] The code is formatted according to the project's coding guidelines and style.
- [X] Necessary documentation has been added or existing documentation has been updated.
- [X] Relevant unit tests have been written and included.
- [X] Relevant integration tests have been written and included.
- [X] The code has been reviewed to ensure its quality.
- [X] The code does not contain any issues flagged by SonarLint.
- [X] Default reviewers have been assigned to this pull request.
- [X] Assignees have been added if necessary.
- [X] Labels have been applied if necessary.
- [X] If an issue is opened and this pull request is related to that issue, it should not be associated with any project
  board or milestone. However, if there is no issue, the pull request should be associated with the appropriate project
  board (BE Board, In Review column) and milestone to track the progress.

### Reviewers

The default project maintainers and team members will be automatically assigned as reviewers for this pull request.

### CODE REVIEW

**Comment by ibrahimdenizz** [comment]
@agitrubard Conflict çözdüldü.

File: src/main/java/com/ays/admin_user/model/mapper/AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * <p>The class provides a static method {@code initialize()} that returns an instance of the generated mapper implementation.
+ * <p>The interface extends the MapStruct interface {@link BaseMapper}, which defines basic mapping methods.
+ * The interface adds no additional mapping methods, but simply defines the types to be used in the mapping process.
+ */
+@Mapper
+public interface AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper extends BaseMapper<AdminUserRegisterApplication, AdminUserRegisterApplicationResponse> {
+
+    @Override
+    @Mapping(target = "user", source = "adminUser")
+    AdminUserRegisterApplicationResponse map(AdminUserRegisterApplication source);
```
Comment (agitrubard) [inline src/main/java/com/ays/admin_user/model/mapper/AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper.java:23]: Listeleme endpointindeki gibi yapabilriiz burayı da

File: src/test/java/com/ays/admin_user/model/dto/request/AdminUserRegisterApplicationListRequestBuilder.java [hunk 2]
```diff
@@ -11,46 +11,46 @@
...
+
+	public AdminUserRegisterApplicationListRequestBuilder withPagination(AysPaging aysPaging) {
+		data.setPagination(aysPaging);
+		return this;
+	}
+
+	public AdminUserRegisterApplicationListRequestBuilder withSort(List<AysSorting> sorting) {
+		data.setSort(sorting);
+		return this;
+	}
```
Comment (agitrubard) [inline src/test/java/com/ays/admin_user/model/dto/request/AdminUserRegisterApplicationListRequestBuilder.java:54]: Sanırım formatlamada bir problem var. Değişiklik yaptığımız sınıflarda formatlama yapabilir miyiz? 

Ek olarak commit atmadan önce otomatik formatlanmasını sağlayabiliriz;
https://www.jetbrains.com/guide/java/tutorials/reformatting-code/reformatting-before-commit/

**Review by agitrubard — CHANGES_REQUESTED** [review]

File: src/main/java/com/ays/admin_user/model/mapper/AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper.java (hunk 1 above)
Comment (ibrahimdenizz) [reply src/main/java/com/ays/admin_user/model/mapper/AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper.java]: Kaldırıldı.

File: src/test/java/com/ays/admin_user/controller/AdminUserRegisterApplicationSystemTest.java [hunk 3]
```diff
@@ -21,15 +21,26 @@
...
     private final AdminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper adminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper = AdminUserRegisterApplicationEntityToAdminUserRegisterApplicationMapper.initialize();
+	private final AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper adminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper = AdminUserRegisterApplicationToAdminUserRegisterApplicationResponseMapper.initialize();
 
 
     private static final String BASE_PATH = "/api/v1/admin";
 
 
+	private void initialize(AdminUserRegisterApplicationEntity mockEntity) {
+		adminUserRegisterApplicationRepository.save(mockEntity);
+	}
```
[the remark this answers is not in the document]
Comment (ibrahimdenizz) [reply src/test/java/com/ays/admin_user/controller/AdminUserRegisterApplicationSystemTest.java]: Format ayarlarım default ayarlardan farklı olduğu için yanlış formatlama yapmış. Projedeki `*` importlarıyla beraber düzeltildi.

**Review by agitrubard — APPROVED** [review]

**Pull request merged**